starters/basic-starter/pages/[...slug].tsx

```diff
@@ -140,3 +143,54 @@ export async function getStaticProps(
     revalidate: 60,
   };
 }
+
+// Generates paths of pages to pre-render with prioritization of menu links.
+async function generatePathsForPrerender(context) {
+  const pathsFromContext = await drupal.getPathsFromContext(
+    CONTENT_TYPES,
+    context,
+  );
+  const pathsFromPages = [];
+  // Get the paths from the pages directory that are pre-rendered by default.
+  fs.readdir(__dirname, (e, files) => {
```

Comment:
I'm not sure this will fly because this would have to be done for nested paths too

Reply:
An alternative approach I would like to propose for this would be to simply rearrange the result of `drupal.getPathsFromContext` based on what we get from `drupal.getMenu('main')`.  On this approach the menu items would be moved to the beginning of the array, and that way they would end up prioritized. With this approach we wouldn't have to worry about duplicate URLs since we would disregard any paths that don't exist in `drupal.getPathsFromContext`.